scootapi/server/job_status.go

```diff
@@ -33,6 +31,8 @@ func GetJobStatus(jobId string, sc s.SagaCoordinator) (*scoot.JobStatus, error)
 		js := scoot.NewJobStatus()
 		js.ID = jobId
 		js.Status = scoot.Status_NOT_STARTED
+		js.TaskStatus = make(map[string]scoot.Status)
+		js.TaskData = make(map[string]*scoot.RunStatus)
```

Comment:
I don't think it has access to what the tasks are - state from GetSagaState here is nil, and the sagaState is what convertSagaStateToJobStatus relies on to update the task statuses.